back/doc/dependency_decisions.yml

```diff
@@ -209,3 +209,9 @@
     :why: 
     :versions: []
     :when: 2024-02-26 08:17:57.582672828 Z
+- - :permit
+  - LGPL
+  - :who: 
+    :why: 
+    :versions: []
+    :when: 2024-11-08 16:55:00.000000000 Z
```

Comment:
I probably missed this last time, but we can't have GPL(-derived) licenses. LGPL is an edge case, but even then, would prefer if we can keep it out. More info [here](https://www.notion.so/govocal/Source-code-licenses-0e56954f330e41ba9bef7045b75eb1c0?pvs=4#c0cbc9a082594c7e93824a5f97cb9d28). Can you look for an alternative?